### PR TITLE
[ADD] [9.0] [stock_quant_reserved_qty_uom]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Stock Quant Reserved Qty UoM
========================

This module allows to display the quantity of a quant in the unit of measure
defined in the reservation move.

### CODE REVIEW

File: stock_quant_reserved_qty_uom/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock Quant Reserved Qty UoM',
+    'version': '8.0.1.0.0',
```
Comment (rousseldenis) [inline stock_quant_reserved_qty_uom/__openerp__.py]: @jbeficent Could you please change version (runbot failed to build) for tests ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_quant_reserved_qty_uom/tests/test_stock_quant_reserved_qty_uom.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestStockQuantReservedQtyUom(TransactionCase):
```
Comment (rousseldenis) [inline stock_quant_reserved_qty_uom/tests/test_stock_quant_reserved_qty_uom.py:8]: @jbeficent It seems that assert test is not executed (if I understand codecoverage result). Can you check ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by JordiBForgeFlow** [comment]
@mreficent can you review the comments made by @rousseldenis on this one?

**Comment by MiquelRForgeFlow** [comment]
Checked and fixed.

**Review by MiquelRForgeFlow — APPROVED** [review]
Tested on local :+1:

**Comment by JordiBForgeFlow** [comment]
@darshan-serpent @lreficent @gmeficent can you review?

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM :+1:

**Review by gmeficent — APPROVED** [review]
Functional review on local, works as described :+1:

**Review by rousseldenis — APPROVED** [review]
:+1:

**Pull request merged**